### PR TITLE
Update validate-lint script

### PR DESCRIPTION
**Pull request by icecrime**
Add a set of newly linted packages, and fix the script.

### CODE REVIEW

File: hack/make/validate-lint [hunk 1]
```diff
@@ -42,7 +47,9 @@ if [ ${#errors[@]} -eq 0 ]; then
 else
 	{
 		echo "Errors from golint:"
-		echo "${errors[@]}"
+		for err in "${errors[@]}"; do
+			echo "$err"
+	    done
```
Comment (tianon) [inline hack/make/validate-lint]: Whitespace here seems a little off... :smile:

**Comment by tianon** [comment]
Nice catch :+1:

**Comment by icecrime** [comment]
Fixed

**Comment by tianon** [comment]
LGTM

**Pull request merged**